scripts/type-generation/allure-reporter-generate-typings.js

```diff
@@ -0,0 +1,38 @@
+#!/usr/bin/env node
+
+const fs = require('fs')
+const path = require('path')
+const { buildCommand } = require('./generate-typings-utils')
+const dox = require('dox')
+
+const gatherCommands = (commandPath) => {
+    const commandContents = fs.readFileSync(commandPath).toString()
+    const commandDocs = dox.parseComments(commandContents)
+    const commands = []
+
+    commandDocs.forEach(command => {
+        if (command.tags) {
+            const nameTag = command.tags.find(tag => tag.type === 'name')
+            if (nameTag && nameTag.string) {
+                commands.push(buildCommand(`function ${nameTag.string}`, command.tags))
+            } else {
+                throw new Error('Missing name tag for command. \n' + command.code)
+            }
+        }
+    })
+
+    return commands
+}
+
+const allureCommands = gatherCommands(path.join(__dirname + '/../../packages/wdio-allure-reporter/src/index.js'))
+
+const templatePath = path.join(__dirname, '../templates', 'allure-reporter.tpl.d.ts')
```

Comment:
This file is missing

Reply:
taking a look. Have I broken something?

Reply:
totally my fault, looks like I forgot to commit and push template file

Reply:
@christian-bromann file looks like this:
```
declare namespace AllureReporter {
    // ... AllureReporter commands ...
}

declare module "@wdio/allure-reporter" {
    export default AllureReporter
}
```

can do PR tomorrow

Reply:
@christian-bromann there was some magic with .gitignore

Can't image how is that webdriver.tpl.d.ts and webdriver.tpl.d.ts are not ignored but allure-reporter.tpl.d.ts is ignored